### PR TITLE
add example in compare function table

### PR DESCRIPTION
**Pull request by tumzied**
### Description
Add  a simple example of sort order  of a compare compare function

### Motivation
Currently, every time I look at the sort order of a  return value compare function it make me question my carrier.  so i have add a simple example to return value.

### CODE REVIEW

**Review by Josh-Cena — COMMENTED** [review]
I don't know if this is more comprehensible than before—especially since there's no guarantee that `a` and `b` would be adjacent. Could you explain the motivation better? The words "after" and "before" seem apparent enough.

**Comment by tumzied** [comment]
While the concept of "sorting a after b" may seem clear and straightforward, it can still be open to interpretation, especially for readers who are less experienced with sorting algorithms or the specific programming language being used. For example, some readers may interpret "sorting a after b" to mean that a should appear immediately after b in the sorted array, rather than to the right of b as intended.

By including the `ex: [b, a]` notation in the documentation for the `compareFn`, readers are provided with a more specific and concrete example of how to achieve the desired sort order. This notation provides an unambiguous way of indicating that a should appear to the right of b in the sorted array, making it easier for readers to understand and apply the `compareFn` correctly.

Additionally, the `ex: [b, a]` notation can help to clarify more complex sorting scenarios where multiple sort criteria are being used. In such scenarios, it may not be immediately obvious how to achieve the desired sort order using only the `compareFn` function. By using the `ex: [b, a]` notation, readers can more easily understand how to modify the `compareFn` to achieve the desired sort order in a way that is consistent with their intended logic.

File: files/en-us/web/javascript/reference/global_objects/array/sort/index.md [hunk 1]
```diff
@@ -69,8 +69,8 @@ elements are sorted according to the return value of the compare function (all
 
 | `compareFn(a, b)` return value | sort order                         |
 | ------------------------------ | ---------------------------------- |
-| > 0                            | sort `a` after `b`                 |
-| < 0                            | sort `a` before `b`                |
+| > 0                            | sort `a` after `b` ; ex: [b, a]    |
+| < 0                            | sort `a` before `b` ; ex: [a, b]   |
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/reference/global_objects/array/sort/index.md:73]:
```suggestion
| > 0                            | sort `a` after `b`, e.g. `[b, a]`    |
| < 0                            | sort `a` before `b`, e.g. `[a, b]`   |
```

**Review by Josh-Cena — COMMENTED** [review]
Thanks. That does help clarify the motivation.

**Review by Josh-Cena — APPROVED** [review]
This looks great to me. Thanks @tumzied and welcome to MDN!

**Pull request merged**